imports/plugins/included/default-theme/client/styles/social.less

```diff
@@ -0,0 +1,3 @@
+.resize {
+font-size: 16px;
```

Comment:
Class name is not expressive and perhaps too generic

This line isn't properly indented